styles/index.css

```diff
@@ -0,0 +1,45 @@
+body {
+    background-color: var(--title-tertiary);
+    display: flex;
+    margin: 0;
+    padding: 0;
+}
+
+@font-face {
+    font-family: 'KoHo';
+    src: url('KoHo-BoldItalic.woff2') format('woff2'),
+        url('KoHo-BoldItalic.woff') format('woff');
+    font-weight: bold;
+    font-style: italic;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'KoHo';
+    src: url('KoHo-Regular.woff2') format('woff2'),
+        url('KoHo-Regular.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+}
+
```

Comment:
we can take away this if we link stylesheet in html healer. all @font face

Reply:
ok